apis/r/R/QueryCondition.R

```diff
@@ -301,20 +307,24 @@ tiledbsoma_empty_query_condition <- function(somactx) {
   return(invisible(query_condition))
 }
 
-# ================================================================
 #' Initialize a 'tiledbsoma_query_condition' object
 #'
-#' Initializes (and possibly allocates) a query condition object using a triplet of
-#' attribute name, comparison value, and operator.  Six types of conditions are supported,
-#' they all take a single scalar comparison argument and attribute to compare against.
-#' At present only integer or numeric attribute comparisons are implemented.
+#' Initializes (and possibly allocates) a query condition object using a
+#' triplet of attribute name, comparison value, and operator.  Six types of
+#' conditions are supported, they all take a single scalar comparison argument
```

Comment:
```suggestion
#' conditions are supported. They all take a single scalar comparison argument
```

(This is a comma splice.)